src/java.base/share/classes/java/lang/Character.java

```diff
@@ -737,10 +737,10 @@ public final String toString() {
      */
     public static final class UnicodeBlock extends Subset {
         /**
-         * 696 - the expected number of entities
+         * 737 - the expected number of entities
```

Comment:
Just a quibble about this comment... it's probably not worth repeating the actual value. But it probably is worth mentioning that the actual value should (or must) match the number of entries added to the map by constructors called from the static initializers in this class. Whenever aliases or new blocks are added, this number must be adjusted.

Reply:
Not a "quibble" at all.  In fact, I thought the same just after I submitted the PR that the number in the comment would be easily overlooked and got stale, which would defy this cleanup. Removed the actual number and put some explanation there.

Reply:
Thanks, looks good.